packages/addons-website/src/components/ImageComponent/ImageComponent-story.js

```diff
@@ -12,25 +12,26 @@ storiesOf('Image Component', module).add(
       text: 'Image Component with Zoom',
     },
   }
-).add(
-    'Image Component with Zoom no Title',
-    () => <ImageComponent caption="This is a caption" fixed="default" zoom="true">
-              <img src="https://loremflickr.com/400/300" alt="alt text" />
-          </ImageComponent>,
-    {
-      info: {
-        text: 'Image Component with Zoom no Title',
-      },
-    }
-  ).add(
-    'Image Component no Zoom',
-    () => <ImageComponent caption="This is a caption" fixed="default">
-              <img src="https://loremflickr.com/400/300" alt="alt text" title="This is the title" />
-          </ImageComponent>,
-    {
-      info: {
-        text: 'Image Component no Zoom',
-      },
-    }
-  )
-  ;
+)
+// .add(
```

Comment:
was this supposed to be commented out?

Reply:
@alisonjoseph this is not a component I was working on but I may have inadvertently commented it out. I have un-commented